common/topology-control/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "nym-topology-control"
+version = "1.1.15"
+authors = ["Dave Hrycyszyn <futurechimp@users.noreply.github.com>"]
```

Comment:
Basically most of these fields should probably use the workspace defaults

Reply:
Apply in 384cafc